src/dune_rules/codept.ml

```diff
@@ -0,0 +1,105 @@
+open Import
+open Dep_gen.Modules_data
+
+let codept_prog ~dir sctx =
+  Super_context.resolve_program sctx ~dir ~loc:None "codept"
```

Comment:
The location will need to come from where the user enabled this

Reply:
I now get the location from the `(using codept 0.1)` stanza.